src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -93,7 +93,7 @@ internal partial class Binder
             }
 
             if (conversion.IsTupleLiteralConversion ||
-                (conversion.Kind == ConversionKind.ImplicitNullable && conversion.UnderlyingConversions[0].IsTupleLiteralConversion))
+                (conversion.IsNullable && conversion.UnderlyingConversions[0].IsTupleLiteralConversion))
```

Comment:
Added.

---
In reply to: [109989611](https://github.com/dotnet/roslyn/pull/18416#discussion_r109989611) [](ancestors = 109989611)